eng/common/pipelines/templates/jobs/prepare-pipelines.yml

```diff
@@ -35,166 +35,181 @@ jobs:
     - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
     # This covers our public repos.
     - ${{ if not(endsWith(parameters.Repository, '-pr'))}}:
-      - script: >
-          $(Pipeline.Workspace)/pipeline-generator/pipeline-generator generate
-          --organization azure-sdk
-          --project public
-          --prefix ${{parameters.Prefix}}
-          --devopspath "\${{parameters.Prefix}}"
-          --path $(System.DefaultWorkingDirectory)/sdk
-          --endpoint Azure
-          --repository ${{parameters.Repository}}
-          --convention ci
-          --agentpool Hosted
-          --branch refs/heads/$(DefaultBranch)
-          --patvar PATVAR
-          --set-managed-variables
-          --debug
-          ${{parameters.CIConventionOptions}}
+      - task: AzureCLI@2
+        inputs:
+          azureSubscription: 'opensource-api-connection'
```

Comment:
Do we know if the identity from this connection has the right level of permissions to create new pipelines? I suspect it doesn't currently so we will need to decide if we want to give it that permission or create a new connection with enough permission for it. I think I'd lean to creating a new one for these pipeline creation pipelines so we can keep it locked down to just those.

Reply:
As per this morning, we know it does.